go.mod

```diff
@@ -10,14 +10,14 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.4
 	github.com/docker/go-units v0.5.0
 	github.com/godbus/dbus/v5 v5.1.0
+	github.com/kolyshkin/capability v0.1.0
```

Comment:
Oh! I saw you work on this repository; should we try get this in either the `opencontainers` or `moby` org?

Reply:
Or maybe the author of https://github.com/syndtr/gocapability is open to transferring maintenance (I think it's largely unmaintained, but could of course be just "moving slow")

Reply:
@thaJeztah I was thinking of opencontainers but I'm not an org admin so can't create new repos here. Maybe moby is a good place; I guess I can transfer the repo.

Reply:
Yeah, would be good; where possible I try to reduce dependencies from personal accounts. No discredit to those authors (you in this case), but not everyone may have familiarity with the author. Having the code in an org also can be more flexible and future-proof (in addition to being able to benefit from such orgs usually being on a (sponsored) enterprise plan, which may come with additional features on GitHub).

Either one would be fine with me; doing a very cursory check on where it's used (and thus _potential_ candidates to switch), I see these;

<img width="318" alt="Screenshot 2024-08-01 at 12 23 42" src="https://github.com/user-attachments/assets/359135a0-57ed-4433-9903-f364d2babafb">


AFAICS it's no longer a direct dependency in any of the projects in the Moby org, but I'd not consider that a blocker for having it in that org (personally). Happy to hear from others though! If we prefer `opencontainers`, I guess we can ask an owner to help transferring.

Reply:
I was talking in another PR about maybe not using the personal account (https://github.com/opencontainers/runc/pull/4362#issuecomment-2263163221). IMHO moby or opencontainers seem fine.

Reply:
Shall we review and merge this or do we want to wait until we sort out the repo ownership?

Reply:
Discussing with maintainers in the Moby call, and it looks like there's no objections against moving it in the moby org, so we may as well try to set that in motion (I can catch up with Kir on that)

Reply:
Yes, let's move this to org repo first.

@caniszczyk can you help me move https://github.com/kolyshkin/capability to under opencontainers org?

Reply:
opened https://github.com/kolyshkin/capability/issues/14

Reply:
In general you need an OCI TOB vote to add projects to opencontainers, I suspect putting it `moby` would be simpler. I'm open to proposing it to the TOB though.